src/main/resources/reference.conf

```diff
@@ -43,4 +43,11 @@ userprofile {
 
 googlecloud {
   priceListUrl: "https://cloudpricingcalculator.appspot.com/static/data/pricelist.json"
+}
+
+trial {
+  spreadsheet {
+    id = ""
+    updateFrequencyMinutes = 0
+  }
```

Comment:
setting some fallback defaults; real conf is in broadinstitute/firecloud-develop#1103